src/systems/systems/src/landing_gear/mod.rs

```diff
@@ -911,13 +925,45 @@ impl LandingGearControlInterfaceUnit {
         self.status = if self
             .gear_system_control
             .init_gear_state(&self.sensor_inputs, !self.is_gear_lever_down)
+        {
+            //If state ok on init let's compute statemachine outputs
+            self.compute_open_doors_output();
+            self.compute_extend_gears_output();
```

Comment:
Will this work for in-flight spawns?

Reply:
It should :). Context is used at object init